ci/tests/general/general_disk.py

```diff
@@ -172,6 +172,52 @@ def append_disk_role(partition, roles_to_add):
                                    roles=roles)
 
     @staticmethod
+    def set_disk_role(partition, roles):
```

Comment:
1. This method calls for another method and does nothing else only making things more complicated
2. Duplicate signature of this method